libr/anal/p/anal_avr.c

```diff
@@ -1,10 +1,11 @@
-/* radare - LGPL - Copyright 2011-2016 - pancake, Roc Valles, condret */
+/* radare - LGPL - Copyright 2011-2016 - pancake, Roc Valles, condret, killabyte */
 
 #if 0
 http://www.atmel.com/images/atmel-0856-avr-instruction-set-manual.pdf
 https://en.wikipedia.org/wiki/Atmel_AVR_instruction_set
 #endif
 
+#define _GNU_SOURCE
```

Comment:
Seems that strcasestr() needs this directive...

radare2 has implemented some helper to avoid using that?